src/http/oauth1.ts

```diff
@@ -94,7 +94,6 @@ export function createOAuth1HttpClient(
     ...config,
     headers: {
       ...config.headers,
-      'Content-Type': 'application/json',
```

Comment:
Did find a bug preventing uploads with react native (both iOS and Android) it was only with oAuth1 because of this header being set (which it doesn't need to). Not a problem for web/node.js it seems 😅